ink/goddess/reward_joi

```diff
@@ -779,20 +783,20 @@ Nod your head when you are finished.
 = cum_comments
     {shuffle:
     - Such a big {player_name}!
-    - Watching my slaves cum always gets me so fucking wet.
+    - The scene of a slaves ejaculation always warms my nether regions.
```

Comment:
I would say 'the sight'... "the scene" doesn't sound quite right.

Reply:
I agree. Great idea!